lib/builderator/tasks.rb

```diff
@@ -74,8 +74,8 @@ def image(profile = :default)
         prepare
 
         invoke Tasks::Packer, :build, [profile], options
-        invoke Tasks::Packer, :remote_tag, [profile], options if options['remote_tag']
         invoke Tasks::Packer, :copy, [profile], options if options['copy']
+        invoke Tasks::Packer, :remote_tag, [profile], options if options['remote_tag']
```

Comment:
We really need a way to disable copying by default.   It can be really slow and we don't (currently!) need it.